play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt

```diff
@@ -1,63 +1,53 @@
 These addons are stat affected items or items that will add functionality to a 
 weapon. Most Attachments can be afixed to any weapon unless otherwise 
-noted. Ask a GM if ou are not sure.
+noted. Ask a GM if you are not sure.
 
 Please note that unless otherwise specified, only one atachment of a given type may
-be used. For instance, one cannot use two red lasers to get a +30% to accuracy, nor 
+be used. For instance, one cannot use two red lasers to get a -30% to miss chance, nor 
 can you use both a reflex sight and a 3.2x scope. 
 
 ====================
 Optics / Aiming Aids
 You may only have 1 scope, unless otherwise stated.
 ====================
 
-Tritium Glow Iron Sights $50
+===== Level 1 Requirement =====
+
+Tritium Glow Iron Sights  $150 
 1x magnification, but allows for much better target aquisition. Can be mounted 
 on 45 degree mounts in addition to a scope in order to negate a minimum 
 engagment distance penalty.
-Miss Chance -5%
-Can be used on any conventional, nonexplosives weapon, E.G. pistols, shotguns, 
+Range Bracket Miss Chance: -0.5
+Can be used on any conventional, nonexplosives weapon. E.G., pistols, shotguns, 
 rifles, etc.
 
-Grenade Launcher Iron Sights $100
+Grenade Launcher Iron Sights  $400
 1x Magnification. Allows better estimation of distances for use with grenade 
 launcher. Folds down for combatibility with magnified optics or dot-sites for
-the primary weapon. Can be used along side other weapon sites, but not the Tritium Glow Iron Sites.
-Underbarrel Grenade Launcher -10% miss chance.
+the primary weapon. Can be used along side other weapon sites, but not the Tritium Glow Iron Sites (If grenade launcher is not an underbarrel attachment, disregard). 
+Underbarrel Grenade Launcher, ranged miss chance: -0.5
 
-1x Dot Sight $130
+1x Dot Sight  $350
 Your basic tube-shaped dot-projection. Dot can be changed from blue-green-red 
 for various lighting situations and terrains.
-Miss Chance -3%
-Range +10m
-
-1x Reflex Sight $150
-A reflex-projected site, comprised of a light source and a single pane of 
-rectangular glass. Allows for smaller F.O.V. blockage. Has a small cross, 
-chevron, or dot as the indicator. Cannot be used on pistols
-Miss Chance -5%
-Range +10m 
-
-1x Pistol Reflex Sight $100
-A reflex-projected site, comprised of a light source and a single pane of 
-rectangular glass. Allows for smaller F.O.V. blockage. Has a small cross, 
-chevron, or dot as the indicator. Can only be used on pistols.
-Miss Chance -5%
-Range +5m
-
-1.2x Hologramic projector $175
+Reduction in Range Bracket Miss Chance by weapon type: -1 for Pistols, -0.5 other.
+R3 Range: +3m

```

Comment:
Only R3 increase? dot sights of all kinds should increase visibility primarily at lower ranges.